router/router.go

```diff
@@ -1994,6 +1993,31 @@ func (rt *HandleT) readAndProcess() int {
 			rt.timeGained += latenciesUsed[job.WorkspaceId]
 
 			rt.MultitenantI.CalculateSuccessFailureCounts(job.WorkspaceId, rt.destName, false, true)
+			/*
+				The next few lines of code is mostly duplicated from commitStatusList function to add
+				stats about the jobs to the reports table
+			*/
+			var parameters JobParametersT
+			err := json.Unmarshal(job.Parameters, &parameters)
+			if err != nil {
+				rt.logger.Error("Unmarshal of job parameters failed. ", string(job.Parameters))
+			}
+			var inPu string
+			if parameters.TransformAt == "processor" {
+				inPu = utilTypes.DEST_TRANSFORMER
+			} else {
+				inPu = utilTypes.EVENT_FILTER
+			}
+
+			eventName := gjson.GetBytes(job.Parameters, "event_name").String()
+			eventType := gjson.GetBytes(job.Parameters, "event_type").String()
+
+			reportMetric := &utilTypes.PUReportedMetric{
+				ConnectionDetails: *utilTypes.CreateConnectionDetail(parameters.SourceID, parameters.DestinationID, parameters.SourceBatchID, parameters.SourceTaskID, parameters.SourceTaskRunID, parameters.SourceJobID, parameters.SourceJobRunID, parameters.SourceDefinitionID, parameters.DestinationDefinitionID, parameters.SourceCategory),
+				PUDetails:         *utilTypes.CreatePUDetails(inPu, utilTypes.ROUTER, true, false),
+				StatusDetail:      utilTypes.CreateStatusDetail(status.JobState, 1, 200, string(status.ErrorResponse), job.EventPayload, eventName, eventType),
```

Comment:
does it make sense to use a 200 HTTP status code for drained events? Using for instance `410 Gone` would be more appropriate.

Reply:
Will do